code/util/gbswd.c

```diff
@@ -273,8 +278,7 @@ int main(int argc,char **argv) {
 			printf("Couldn't create %s\n",outname);
 			return 3;
 		}
-		void be32(char *p, int v) {p[3] = v;p[2] = v>>8;p[1] = v>>16;p[0] = v>>24;}
```

Comment:
NO leave the code where it is

Reply:
Ditto

Reply:
having a function inside a function like this is a non-standard GNU extension that doesn't work without GCC. MacOS doesn't have GCC anymore (due to licensing issues) so this can't be helped.